src/main/java/dao/FileWriterImpl.java

```diff
@@ -0,0 +1,24 @@
+package dao;
+
+import java.io.BufferedWriter;
```

Comment:
Remember to use try-with-resources with classes that implement the AutoCloseable interface. Good job using it with `BufferedWriter` here.